msgid ""
msgstr ""
"Project-Id-Version: 'addonPackager' '1.1'\n"
"Report-Msgid-Bugs-To: 'nvda-translations@groups.io'\n"
"POT-Creation-Date: 2022-10-24 12:38+0200\n"
"PO-Revision-Date: 2022-11-17 13:34+0100\n"
"Last-Translator: Rémy Ruiz <remyruiz@gmail.com>\n"
"Language-Team: \n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 3.1\n"
"X-Poedit-SourceCharset: UTF-8\n"

#. Add-on summary, usually the user visible name of the addon.
#. Translators: Summary for this add-on to be shown on installation and add-on information.
#: buildVars.py:23 addon\globalPlugins\addonPackager\__init__.py:67
#: addon\globalPlugins\addonPackager\main.py:29
msgid "Utilidades para los complementos de NVDA"
msgstr "Utilitaires pour les extensions de NVDA"

#. Add-on description
#. Translators: Long description to be shown for this add-on on add-on information from add-ons manager
#: buildVars.py:26
msgid ""
"Conjunto de utilidades para un manejo fácil de complementos para NVDA.\n"
"\n"
"* Empaquetador de complementos.\n"
"* Instalador múltiple de complementos.\n"
"* Desinstala complementos.\n"
"* Habilita / deshabilita complementos.\n"
"* Modificador de manifiestos, instalados y desde archivo no instalado.\n"
"* Hacer / restaurar copias de seguridad de la configuración de NVDA.\n"
"* Documentación de complementos.\n"
"\n"
"Todas las áreas de fácil manejo y para ahorrar tiempo respecto al gestor de "
"complementos nativo de NVDA."
msgstr ""
"Ensemble d'utilitaires pour une gestion facile des extensions pour NVDA.\n"
"\n"
"* Empaqueteuse d'extensions.\n"
"* Installateur multiple d'extensions.\n"
"* Désinstalle des extensions.\n"
"* Active / désactive les extensions.\n"
"* Modificateur des manifestes, installé et à partir de fichier non "
"installé.\n"
"* Faire / restaurer des sauvegardes de la configuration de NVDA.\n"
"* Documentation des extensions.\n"
"\n"
"Toutes les zones de facile manipulation et pour gagner du temps concernant "
"le Gestionnaire d'Extensions natif de NVDA."

#: addon\globalPlugins\addonPackager\__init__.py:35
msgid "&Utilidades para los complementos de NVDA"
msgstr "&Utilitaires pour les extensions de NVDA"

#: addon\globalPlugins\addonPackager\__init__.py:52
msgid ""
"Hay una acción pendiente de reiniciar NVDA.\n"
"\n"
"Por motivos de seguridad el complemento no podrá ser usado hasta que no "
"reinicie.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Il y a une action en attente pour redémarrer NVDA.\n"
"\n"
"Pour des raisons de sécurité, l'extension ne peut être utilisée jusqu'à ce "
"qu'elle redémarre.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\__init__.py:57
msgid "Pregunta"
msgstr "Question"

#: addon\globalPlugins\addonPackager\__init__.py:67
msgid "Muestra la ventana de Utilidades para los complementos de NVDA"
msgstr ""
"Affiche la fenêtre de l'extension Utilitaires pour les extensions de NVDA"

#: addon\globalPlugins\addonPackager\__init__.py:77
#: addon\globalPlugins\addonPackager\__init__.py:79
msgid ""
"Ya hay una instancia de Utilidades para los complementos de NVDA abierta."
msgstr ""
"Il existe déjà une instance de l'extension Utilitaires pour les extensions "
"de NVDA ouverte."

#: addon\globalPlugins\addonPackager\__init__.py:77
#: addon\globalPlugins\addonPackager\main.py:406
msgid "Información"
msgstr "Informations"

#: addon\globalPlugins\addonPackager\main.py:47
msgid "Empaquetador de complementos"
msgstr "Empaqueteuse d'extensions"

#: addon\globalPlugins\addonPackager\main.py:51
#: addon\globalPlugins\addonPackager\main.py:74
#: addon\globalPlugins\addonPackager\main.py:94
#: addon\globalPlugins\addonPackager\main.py:201
msgid "&Lista de complementos:"
msgstr "&Liste des extensions:"

#: addon\globalPlugins\addonPackager\main.py:60
#: addon\globalPlugins\addonPackager\main.py:83
#: addon\globalPlugins\addonPackager\main.py:103
#: addon\globalPlugins\addonPackager\main.py:129
#: addon\globalPlugins\addonPackager\main.py:170
msgid "&Selección"
msgstr "&Sélection"

#: addon\globalPlugins\addonPackager\main.py:63
msgid "&Generar"
msgstr "&Générer"

#: addon\globalPlugins\addonPackager\main.py:67
msgid "Instalador múltiple"
msgstr "Installateur multiple"

#: addon\globalPlugins\addonPackager\main.py:71
msgid "&Seleccione un directorio con complementos a instalar..."
msgstr "&Sélectionner un répertoire avec les extensions à installer..."

#: addon\globalPlugins\addonPackager\main.py:86
msgid "&Instalar"
msgstr "&Installer"

#: addon\globalPlugins\addonPackager\main.py:90
msgid "Desinstala complementos"
msgstr "Désinstalle des extensions"

#: addon\globalPlugins\addonPackager\main.py:106
msgid "&Desinstalar"
msgstr "&Désinstaller"

#: addon\globalPlugins\addonPackager\main.py:110
msgid "Habilita / deshabilita complementos"
msgstr "Active / désactive les extensions"

#: addon\globalPlugins\addonPackager\main.py:114
msgid "&Lista de complementos habilitados:"
msgstr "&Liste des extensions activées:"

#: addon\globalPlugins\addonPackager\main.py:120
msgid "&Lista de complementos deshabilitados:"
msgstr "&Liste des extensions désactivées:"

#: addon\globalPlugins\addonPackager\main.py:132
#: addon\globalPlugins\addonPackager\main.py:173
msgid "&Procesar"
msgstr "&Traiter"

#: addon\globalPlugins\addonPackager\main.py:136
msgid "Modificador de manifiestos"
msgstr "Modificateur des manifestes"

#: addon\globalPlugins\addonPackager\main.py:140
msgid "&Lista de complementos instalados:"
msgstr "&Liste des extensions installées:"

#: addon\globalPlugins\addonPackager\main.py:149
msgid "Seleccione versión Mayor:"
msgstr "Sélectionner la version Majeure:"

#: addon\globalPlugins\addonPackager\main.py:155
msgid "Seleccione versión Menor:"
msgstr "Sélectionner la version Mineure:"

#: addon\globalPlugins\addonPackager\main.py:161
msgid "Seleccione una revisión:"
msgstr "Sélectionner une révision:"

#: addon\globalPlugins\addonPackager\main.py:177
msgid "Hacer / restaurar copias de seguridad"
msgstr "Faire / restaurer des sauvegardes"

#: addon\globalPlugins\addonPackager\main.py:181
msgid "&Hacer copias de seguridad:"
msgstr "&Faire des sauvegardes:"

#: addon\globalPlugins\addonPackager\main.py:190
msgid "&Crear copia de seguridad"
msgstr "&Créer une sauvegarde"

#: addon\globalPlugins\addonPackager\main.py:193
msgid "&Restaurar copia de seguridad"
msgstr "&Restaurer une sauvegarde"

#: addon\globalPlugins\addonPackager\main.py:197
msgid "Documentación de complementos"
msgstr "Documentation des extensions"

#: addon\globalPlugins\addonPackager\main.py:207
msgid "&Abrir documentación del complemento"
msgstr "&Ouvrir la documentation de l'extension"

#: addon\globalPlugins\addonPackager\main.py:213
msgid "&Estado:"
msgstr "&Etat:"

#: addon\globalPlugins\addonPackager\main.py:225
msgid "&Aceptar"
msgstr "&OK"

#: addon\globalPlugins\addonPackager\main.py:228
msgid "&Cancelar"
msgstr "&Annuler"

#: addon\globalPlugins\addonPackager\main.py:231
msgid "Cerrar"
msgstr "Fermer"

#: addon\globalPlugins\addonPackager\main.py:315
#: addon\globalPlugins\addonPackager\main.py:581
#: addon\globalPlugins\addonPackager\main.py:625
#: addon\globalPlugins\addonPackager\main.py:862
msgid "Directorio Diccionarios"
msgstr "Répertoire Dictionnaires"

#: addon\globalPlugins\addonPackager\main.py:319
#: addon\globalPlugins\addonPackager\main.py:585
#: addon\globalPlugins\addonPackager\main.py:629
#: addon\globalPlugins\addonPackager\main.py:864
msgid "Directorio Perfiles"
msgstr "Répertoire Profils"

#: addon\globalPlugins\addonPackager\main.py:323
#: addon\globalPlugins\addonPackager\main.py:589
#: addon\globalPlugins\addonPackager\main.py:633
#: addon\globalPlugins\addonPackager\main.py:866
msgid "Directorio Scratchpad"
msgstr "Répertoire Bloc-notes du Développeur"

#: addon\globalPlugins\addonPackager\main.py:327
#: addon\globalPlugins\addonPackager\main.py:593
#: addon\globalPlugins\addonPackager\main.py:637
#: addon\globalPlugins\addonPackager\main.py:868
msgid "Fichero configuración disparadores de perfiles"
msgstr "Fichier de configuration déclencheurs de profils"

#: addon\globalPlugins\addonPackager\main.py:330
#: addon\globalPlugins\addonPackager\main.py:596
#: addon\globalPlugins\addonPackager\main.py:640
#: addon\globalPlugins\addonPackager\main.py:870
msgid "Fichero de configuración gestos de entrada"
msgstr "Fichier de configuration gestes de commandes"

#: addon\globalPlugins\addonPackager\main.py:333
#: addon\globalPlugins\addonPackager\main.py:599
#: addon\globalPlugins\addonPackager\main.py:643
#: addon\globalPlugins\addonPackager\main.py:872
msgid "Fichero de configuración de NVDA"
msgstr "Fichier de configuration NVDA"

#: addon\globalPlugins\addonPackager\main.py:376
#: addon\globalPlugins\addonPackager\main.py:421
#: addon\globalPlugins\addonPackager\main.py:440
#: addon\globalPlugins\addonPackager\main.py:460
#: addon\globalPlugins\addonPackager\main.py:464
#: addon\globalPlugins\addonPackager\main.py:468
#: addon\globalPlugins\addonPackager\main.py:490
msgid "&Seleccionar todo"
msgstr "&Sélectionner tout"

#: addon\globalPlugins\addonPackager\main.py:377
#: addon\globalPlugins\addonPackager\main.py:422
#: addon\globalPlugins\addonPackager\main.py:441
#: addon\globalPlugins\addonPackager\main.py:461
#: addon\globalPlugins\addonPackager\main.py:465
#: addon\globalPlugins\addonPackager\main.py:469
#: addon\globalPlugins\addonPackager\main.py:491
msgid "&Deseleccionar todo"
msgstr "&Désélectionner tout"

#: addon\globalPlugins\addonPackager\main.py:383
msgid "Tiene que seleccionar un complemento para continuar."
msgstr "Vous devez sélectionner une extension pour continuer."

#: addon\globalPlugins\addonPackager\main.py:383
#: addon\globalPlugins\addonPackager\main.py:428
#: addon\globalPlugins\addonPackager\main.py:447
#: addon\globalPlugins\addonPackager\main.py:478
#: addon\globalPlugins\addonPackager\main.py:506
#: addon\globalPlugins\addonPackager\main.py:747
#: addon\globalPlugins\addonPackager\main.py:879
msgid "Error"
msgstr "Erreur"

#. Botón selecciona directorio con complementos instalador.
#: addon\globalPlugins\addonPackager\main.py:386
#: addon\globalPlugins\addonPackager\main.py:400
msgid "Seleccione un directorio:"
msgstr "Sélectionner un répertoire:"

#: addon\globalPlugins\addonPackager\main.py:406
msgid "No se encontraron complementos en el directorio que selecciono."
msgstr ""
"Aucune extension n'a été trouvée dans le répertoire que vous avez sélectionné"

#: addon\globalPlugins\addonPackager\main.py:428
#: addon\globalPlugins\addonPackager\main.py:447
#: addon\globalPlugins\addonPackager\main.py:478
#: addon\globalPlugins\addonPackager\main.py:747
msgid "Necesita elegir al menos un complemento para continuar."
msgstr "Vous devez choisir au moins une extension pour continuer."

#: addon\globalPlugins\addonPackager\main.py:466
msgid "Complementos &habilitados"
msgstr "Extensions &activées"

#: addon\globalPlugins\addonPackager\main.py:470
msgid "Complementos &deshabilitados"
msgstr "Extensions &désactivées"

#: addon\globalPlugins\addonPackager\main.py:496
msgid "Procesar &instalados"
msgstr "Traiter &installées"

#: addon\globalPlugins\addonPackager\main.py:497
msgid "Procesar un archivo de &complemento"
msgstr "Traiter un fichier d'&extension"

#: addon\globalPlugins\addonPackager\main.py:506
#: addon\globalPlugins\addonPackager\main.py:879
msgid ""
"Necesita elegir al menos un elemento de copia de seguridad de la lista para "
"continuar."
msgstr ""
"Vous devez choisir au moins un élément de sauvegarde de la liste pour "
"continuer."

#. Botón restaurar copia de seguridad
#: addon\globalPlugins\addonPackager\main.py:512
#: addon\globalPlugins\addonPackager\main.py:528
msgid "Archivo de copia de seguridad de NVDA (*.nvda-backup)|*.nvda-backup"
msgstr "Fichier de sauvegarde NVDA (*.nvda-backup)|*.nvda-backup"

#: addon\globalPlugins\addonPackager\main.py:513
msgid "Guardar la copia de seguridad en..."
msgstr "Enregistrer la sauvegarde sous..."

#: addon\globalPlugins\addonPackager\main.py:529
msgid "Seleccione un archivo de copia de seguridad de NVDA"
msgstr "Sélectionner un fichier de sauvegarde NVDA"

#. Manifiestos procesar fichero.
#: addon\globalPlugins\addonPackager\main.py:761
msgid "Complemento de NVDA (*.nvda-addon)|*.nvda-addon"
msgstr "Extension NVDA (*.nvda-addon)|*.nvda-addon"

#: addon\globalPlugins\addonPackager\main.py:762
msgid "Seleccione un complemento de NVDA"
msgstr "Sélectionner une extension de NVDA"

#: addon\globalPlugins\addonPackager\main.py:823
msgid "Restaurar copia de seguridad de NVDA"
msgstr "Restaurer une sauvegarde NVDA"

#: addon\globalPlugins\addonPackager\main.py:829
msgid "&Contenido de la copia de seguridad:"
msgstr "&Contenu de la sauvegarde:"

#: addon\globalPlugins\addonPackager\main.py:838
msgid "&Restaurar"
msgstr "&Restaurer"

#: addon\globalPlugins\addonPackager\main.py:841
msgid "&Cerrar"
msgstr "&Fermer"

#: addon\globalPlugins\addonPackager\utilidades.py:162
msgid "Creando el complemento: {} {}"
msgstr "Création en cours de l'extension: {} {}"

#: addon\globalPlugins\addonPackager\utilidades.py:166
msgid ""
"Se generaron correctamente todos los complementos.\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"Toutes les extensions ont été générées avec succès.\n"
"\n"
"Appuyer sur OK pour continuer."

#: addon\globalPlugins\addonPackager\utilidades.py:172
#: addon\globalPlugins\addonPackager\utilidades.py:254
#: addon\globalPlugins\addonPackager\utilidades.py:304
#: addon\globalPlugins\addonPackager\utilidades.py:436
#: addon\globalPlugins\addonPackager\utilidades.py:490
#: addon\globalPlugins\addonPackager\utilidades.py:553
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"Vuelva a intentarlo."
msgstr ""
"Il y a eu une erreur inattendue. \n"
"\n"
"Erreur: {}\n"
"\n"
"Réessayer."

#: addon\globalPlugins\addonPackager\utilidades.py:195
msgid "Instalando el complemento: {}"
msgstr "Installation en cours de l'extension: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:218
msgid ""
"Se completo la instalación correctamente.\n"
"\n"
"NVDA necesita reiniciarse para aplicar las instalaciones.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"L'installation a terminée avec succès.\n"
"\n"
"NVDA doit être redémarré pour que ces installations prennent effet.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:228
msgid ""
"No se pudo instalar el complemento.\n"
"\n"
"Fallo de compatibilidad.\n"
"\n"
"Busque un complemento compatible."
msgstr ""
"Impossible d'installer l'extension.\n"
"\n"
"Échec de la compatibilité.\n"
"\n"
"Rechercher une extension compatible."

#: addon\globalPlugins\addonPackager\utilidades.py:239
msgid ""
"Se completo la instalación correctamente.\n"
"\n"
"Pero hay complementos que no se pudieron instalar.\n"
"\n"
"Los siguientes complementos son incompatibles, busque una versión "
"compatible:\n"
"\n"
"{}\n"
"\n"
"NVDA necesita reiniciarse para aplicar las instalaciones.\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"L'installation a terminée avec succès.\n"
"\n"
"Mais il y a des extensions qui ne pouvaient pas être installées.\n"
"\n"
"Les extensions suivantes sont incompatibles, rechercher une version "
"compatible:\n"
"\n"
"{}\n"
"\n"
"NVDA doit être redémarré pour que ces installations prennent effet.\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:263
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se instalaron y NVDA necesita reiniciar "
"para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Il y a eu une erreur inattendue. \n"
"\n"
"Erreur: {}\n"
"\n"
"Cependant, il y a eu des extensions qui ont été installées et NVDA doit être "
"redémarré pour que ces changements prennent effet.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:288
msgid "Desinstalando el complemento: {}"
msgstr "Désinstallation en cours de l'extension: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:295
msgid ""
"Se completo la desinstalación correctamente.\n"
"\n"
"NVDA necesita reiniciarse para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"La désinstallation a terminée avec succès.\n"
"\n"
"NVDA doit être redémarré pour que ces changements prennent effet.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:313
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se desinstalaron y NVDA necesita reiniciar "
"para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Il y a eu une erreur inattendue. \n"
"\n"
"Erreur: {}\n"
"\n"
"Cependant, il y a eu des extensions qui ont été désinstallées et NVDA doit "
"être redémarré pour que ces changements prennent effet.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:361
msgid "Buscando complementos en {}..."
msgstr "Recherche en cours des extensions dans {}..."

#: addon\globalPlugins\addonPackager\utilidades.py:365
msgid "No hay complementos compatibles en el directorio."
msgstr "Il n'y a pas d'extensions compatibles dans le répertoire."

#: addon\globalPlugins\addonPackager\utilidades.py:369
msgid ""
"Se completo con éxito la extracción de información de los complementos.\n"
"\n"
"No obstante, el directorio que introdujo contiene complementos no "
"compatibles.\n"
"\n"
"* Complementos con fallo en el manifiesto: {}.\n"
"\n"
"{}\n"
"\n"
"* Complementos no compatibles con la api actual: {}.\n"
"\n"
"{}\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"L'extraction des informations des extensions est terminée avec succès.\n"
"\n"
"Cependant, le répertoire qui a été introduit contient des extensions non "
"compatibles.\n"
"\n"
"* Extensions avec échec dans le manifeste: {}.\n"
"\n"
"{}\n"
"\n"
"* Extensions non compatibles avec l'API actuelle: {}.\n"
"\n"
"{}\n"
"\n"
"Appuyer sur OK pour continuer."

#: addon\globalPlugins\addonPackager\utilidades.py:385
msgid ""
"Se completo con éxito la extracción de información de los complementos.\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"L'extraction des informations des extensions est terminée avec succès.\n"
"\n"
"Appuyer sur OK pour continuer."

#: addon\globalPlugins\addonPackager\utilidades.py:408
msgid "Deshabilitando complementos..."
msgstr "Désactivation des extensions..."

#: addon\globalPlugins\addonPackager\utilidades.py:418
msgid "Habilitando complementos..."
msgstr "Activation des extensions..."

#: addon\globalPlugins\addonPackager\utilidades.py:427
msgid ""
"Se completo el proceso de habilitar y deshabilitar correctamente.\n"
"\n"
"NVDA necesita reiniciarse para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Le processus d'activation et de désactivation a terminée avec succès.\n"
"\n"
"NVDA doit être redémarré pour que ces changements prennent effet.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:445
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se habilitaron o deshabilitaron y NVDA "
"necesita reiniciar para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Il y a eu une erreur inattendue. \n"
"\n"
"Erreur: {}\n"
"\n"
"Cependant, il y a eu des extensions qui ont été activées ou désactivées et "
"NVDA doit être redémarré pour que ces changements prennent effet.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:474
msgid "Cambiando el manifiesto al complemento: {}"
msgstr "Changement du manifeste en cours à l'extension: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:481
msgid ""
"Se cambiaron los manifiestos correctamente.\n"
"\n"
"Para que la acción tenga efecto NVDA tiene que reiniciarse.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Les manifestes ont été changés avec succès.\n"
"\n"
"Pour que ces changements prennent effet NVDA doit redémarrer.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:499
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se les cambio el manifiesto y NVDA "
"necesita reiniciar para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Il y a eu une erreur inattendue. \n"
"\n"
"Erreur: {}\n"
"\n"
"Cependant, il y a eu des extensions où ont été changés les manifestes et "
"NVDA doit être redémarré pour que ces changements prennent effet.\n"
"\n"
"Voulez-vous redémarrer NVDA maintenant ?"

#: addon\globalPlugins\addonPackager\utilidades.py:528
msgid "Trabajando en el complemento: {}"
msgstr "Travail en cours dans l'extension: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:543
msgid ""
"Se cambio correctamente el manifiesto.\n"
"\n"
"Se genero un complemento con el manifiesto cambiado en:\n"
"\n"
"{}\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"Le manifeste a été changé avec succès.\n"
"\n"
"Il a été généré une extension avec le manifeste changé dans:\n"
"\n"
"{}\n"
"\n"
"Appuyer sur OK pour continuer."

#. Diccionario
#: addon\globalPlugins\addonPackager\utilidades.py:581
msgid "Añadiendo el directorio de diccionarios..."
msgstr "Ajout du Répertoire Dictionnaires..."

#: addon\globalPlugins\addonPackager\utilidades.py:587
msgid ""
"No se pudo agregar el directorio de diccionarios a la copia de seguridad."
msgstr "Impossible d'ajouter le Répertoire Dictionnaires à la sauvegarde."

#. Perfiles
#: addon\globalPlugins\addonPackager\utilidades.py:589
msgid "Añadiendo el directorio de Perfiles..."
msgstr "Ajout du Répertoire Profils..."

#: addon\globalPlugins\addonPackager\utilidades.py:595
msgid "No se pudo agregar el directorio de Perfiles a la copia de seguridad."
msgstr "Impossible d'ajouter le Répertoire Profils à la sauvegarde."

#. Scratchpad
#: addon\globalPlugins\addonPackager\utilidades.py:597
msgid "Añadiendo el directorio de Scratchpad..."
msgstr "Ajout du Répertoire Bloc-notes du Développeur..."

#: addon\globalPlugins\addonPackager\utilidades.py:603
msgid "No se pudo agregar el directorio de Scratchpad a la copia de seguridad."
msgstr ""
"Impossible d'ajouter le Répertoire Bloc-notes du Développeur à la sauvegarde."

#. Fichero configuración disparadores de perfiles
#: addon\globalPlugins\addonPackager\utilidades.py:606
msgid "Añadiendo el Fichero configuración disparadores de perfiles..."
msgstr "Ajout du Fichier de configuration déclencheurs de profils..."

#: addon\globalPlugins\addonPackager\utilidades.py:613
#: addon\globalPlugins\addonPackager\utilidades.py:615
msgid ""
"No se pudo agregar el Fichero configuración disparadores de perfiles a la "
"copia de seguridad."
msgstr ""
"Impossible d'ajouter le Fichier de configuration déclencheurs de profils à "
"la sauvegarde."

#. Fichero de configuración gestos de entrada
#: addon\globalPlugins\addonPackager\utilidades.py:618
msgid "Añadiendo el Fichero de configuración gestos de entrada..."
msgstr "Ajout du Fichier de configuration gestes de commandes..."

#: addon\globalPlugins\addonPackager\utilidades.py:625
#: addon\globalPlugins\addonPackager\utilidades.py:627
msgid ""
"No se pudo agregar el Fichero de configuración gestos de entrada a la copia "
"de seguridad."
msgstr ""
"Impossible d'ajouter le Fichier de configuration gestes de commandes à la "
"sauvegarde."

#. Fichero de configuración de NVDA
#: addon\globalPlugins\addonPackager\utilidades.py:630
msgid "Añadiendo el Fichero de configuración de NVDA..."
msgstr "Ajout du Fichier de configuration NVDA..."

#: addon\globalPlugins\addonPackager\utilidades.py:637
#: addon\globalPlugins\addonPackager\utilidades.py:639
msgid ""
"No se pudo agregar el Fichero de configuración de NVDA a la copia de "
"seguridad."
msgstr "Impossible d'ajouter le Fichier de configuration NVDA à la sauvegarde."

#: addon\globalPlugins\addonPackager\utilidades.py:652
msgid ""
"Se creo la copia de seguridad correctamente en:\n"
"\n"
"{}\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"La sauvegarde a été créée avec succès dans:\n"
"\n"
"{}\n"
"\n"
"Appuyez sur OK pour continuer."

#: addon\globalPlugins\addonPackager\utilidades.py:665
msgid ""
"Se produjo el siguiente error:\n"
"\n"
"{}\n"
"\n"
"No se pudo crear la copia de seguridad."
msgstr ""
"L'erreur suivante est survenue:\n"
"\n"
"{}\n"
"\n"
"Impossible de créer la sauvegarde."

#: addon\globalPlugins\addonPackager\utilidades.py:673
msgid ""
"Se produjeron los siguientes errores:\n"
"\n"
"{}\n"
"\n"
"No se pudo crear la copia de seguridad."
msgstr ""
"Les erreurs suivantes sont survenues:\n"
"\n"
"{}\n"
"\n"
"Impossible de créer la sauvegarde."

#. Diccionario
#: addon\globalPlugins\addonPackager\utilidades.py:699
msgid "Restaurando el directorio de diccionarios..."
msgstr "Restauration du Répertoire Dictionnaires..."

#: addon\globalPlugins\addonPackager\utilidades.py:708
#: addon\globalPlugins\addonPackager\utilidades.py:710
msgid "No se pudo restaurar el directorio de diccionarios."
msgstr "Impossible de restaurer le Répertoire Dictionnaires."

#. Perfiles
#: addon\globalPlugins\addonPackager\utilidades.py:712
msgid "Restaurando el directorio de Perfiles..."
msgstr "Restauration du Répertoire Profils..."

#: addon\globalPlugins\addonPackager\utilidades.py:721
#: addon\globalPlugins\addonPackager\utilidades.py:723
msgid "No se pudo restaurar el directorio de Perfiles."
msgstr "Impossible de restaurer le Répertoire Profils."

#. Scratchpad
#: addon\globalPlugins\addonPackager\utilidades.py:726
msgid "Restaurando el directorio de Scratchpad..."
msgstr "Restauration du Répertoire Bloc-notes du Développeur..."

#: addon\globalPlugins\addonPackager\utilidades.py:735
#: addon\globalPlugins\addonPackager\utilidades.py:737
msgid "No se pudo restaurar el directorio de Scratchpad."
msgstr "Impossible de restaurer le Répertoire Bloc-notes du Développeur."

#. Fichero configuración disparadores de perfiles
#: addon\globalPlugins\addonPackager\utilidades.py:740
msgid "Restaurando el Fichero configuración disparadores de perfiles..."
msgstr "Restauration du Fichier de configuration déclencheurs de profils..."

#: addon\globalPlugins\addonPackager\utilidades.py:754
#: addon\globalPlugins\addonPackager\utilidades.py:756
#: addon\globalPlugins\addonPackager\utilidades.py:758
msgid "No se pudo restaurar el Fichero configuración disparadores de perfiles."
msgstr ""
"Impossible de restaurer le Fichier de configuration déclencheurs de profils."

#. Fichero configuración gestos de entrada
#: addon\globalPlugins\addonPackager\utilidades.py:761
msgid "Restaurando el Fichero configuración gestos de entrada..."
msgstr "Restauration du Fichier de configuration gestes de commandes..."

#: addon\globalPlugins\addonPackager\utilidades.py:775
#: addon\globalPlugins\addonPackager\utilidades.py:777
msgid "No se pudo restaurar el Fichero configuración de gestos de entrada."
msgstr ""
"Impossible de restaurer le Fichier de configuration gestes de commandes."

#: addon\globalPlugins\addonPackager\utilidades.py:779
msgid "No se pudo restaurar el Fichero configuración gestos de entrada."
msgstr ""
"Impossible de restaurer le Fichier de configuration gestes de commandes."

#. Fichero configuración de NVDA
#: addon\globalPlugins\addonPackager\utilidades.py:782
msgid "Restaurando el Fichero configuración de NVDA..."
msgstr "Restauration du Fichier de configuration NVDA..."

#: addon\globalPlugins\addonPackager\utilidades.py:796
#: addon\globalPlugins\addonPackager\utilidades.py:798
#: addon\globalPlugins\addonPackager\utilidades.py:800
msgid "No se pudo restaurar el Fichero configuración de NVDA."
msgstr "Impossible de restaurer le Fichier de configuration NVDA."

#: addon\globalPlugins\addonPackager\utilidades.py:808
msgid ""
"La restauración fue un éxito.\n"
"\n"
"Al restaurar la configuración de NVDA es necesario reiniciar el lector.\n"
"\n"
"Pulse aceptar o cerrar para reiniciar."
msgstr ""
"La restauration a été réalisée avec succès.\n"
"\n"
"Lors de la restauration de la configuration NVDA, il est nécessaire de "
"redémarrer le lecteur.\n"
"\n"
"Appuyez sur OK ou Fermer pour redémarrer."

#: addon\globalPlugins\addonPackager\utilidades.py:818
msgid ""
"Se produjo el siguiente error:\n"
"\n"
"{}\n"
"\n"
"No se pudo restaurar toda la copia de seguridad.\n"
"\n"
"Pero hay elementos que si pudieron ser restaurados.\n"
"\n"
"Al restaurar la configuración de NVDA es necesario reiniciar el lector.\n"
"\n"
"Pulse cancelar o cerrar para reiniciar."
msgstr ""
"L'erreur suivante s'est produite:\n"
"\n"
"{}\n"
"\n"
"Impossible de restaurer  toute la sauvegarde.\n"
"\n"
"Mais il y a des éléments que si ont pu être restaurés.\n"
"\n"
"Lors de la restauration de la configuration NVDA, il est nécessaire de "
"redémarrer le lecteur.\n"
"\n"
"Appuyez sur Annuler ou Fermer pour redémarrer."

#: addon\globalPlugins\addonPackager\utilidades.py:832
msgid ""
"Se produjo el siguiente error:\n"
"\n"
"{}\n"
"\n"
"No se pudo restaurar la copia de seguridad.\n"
"\n"
"Pulse cancelar o cerrar para continuar."
msgstr ""
"L'erreur suivante s'est produite:\n"
"\n"
"{}\n"
"\n"
"Impossible de restaurer la sauvegarde.\n"
"\n"
"Appuyez sur Annuler ou Fermer pour continuer."

#: addon\globalPlugins\addonPackager\utilidades.py:843
msgid ""
"Se produjeron los siguientes errores:\n"
"\n"
"{}\n"
"\n"
"No se pudo restaurar toda la copia de seguridad.\n"
"\n"
"Pero hay elementos que si pudieron ser restaurados.\n"
"\n"
"Al restaurar la configuración de NVDA es necesario reiniciar el lector.\n"
"\n"
"Pulse cancelar o cerrar para reiniciar."
msgstr ""
"Les erreurs suivantes sont survenues:\n"
"\n"
"{}\n"
"\n"
"Impossible de restaurer  toute la sauvegarde.\n"
"\n"
"Mais il y a des éléments que si ont pu être restaurés.\n"
"\n"
"Lors de la restauration de la configuration NVDA, il est nécessaire de "
"redémarrer le lecteur.\n"
"\n"
"Appuyez sur Annuler ou Fermer pour redémarrer."

#: addon\globalPlugins\addonPackager\utilidades.py:857
msgid ""
"Se produjeron los siguientes errores:\n"
"\n"
"{}\n"
"\n"
"No se pudo restaurar la copia de seguridad.\n"
"\n"
"Pulse cancelar o cerrar para continuar."
msgstr ""
"Les erreurs suivantes sont survenues:\n"
"\n"
"{}\n"
"\n"
"Impossible de restaurer la sauvegarde.\n"
"\n"
"Appuyez sur Annuler ou Fermer pour continuer."
